container/shim/src/config.js

```diff
@@ -15,7 +15,9 @@ export const NODE_OPERATOR_EMAIL = process.env.NODE_OPERATOR_EMAIL || error('NOD
 export const IPFS_GATEWAY_ORIGIN = process.env.IPFS_GATEWAY_ORIGIN || 'https://ipfs.io'
 export const TESTING_CID = 'QmXjYBY478Cno4jzdCcPy4NcJYFrwHZ51xaCP8vUwN9MGm'
 export const nodeId = await readFile(NODE_ID_FILE_PATH, 'utf-8').catch(() => false) || createNodeId()
-export const L2_FIRE_AND_FORGET = SATURN_NETWORK === 'test'
+export const L2_FIRE_AND_FORGET = process.env.L2_FIRE_AND_FORGET
+  ? process.env.L2_FIRE_AND_FORGET === 'true'
+  : SATURN_NETWORK === 'test'
```

Comment:
this is a it awkward, and unflexible. What do you think about adding a `config` option for creating that express app, which lets us override values without needing to change `process.env`?

Reply:
I am not exactly sure what you mean by this, do you have an example?

Reply:
```js
// in tests:

const app = createApp({
  l2FireAndForget: true,
  filWalletAddress: 'test'
})

// in production

const app = createApp({
  l2FireAndForget: SATURN_NETWORK === 'test',
  filWalletAddress: process.env.FIL_WALLET_ADDRESS || error('oh no')
})
```

Reply:
so basically we let `config.js` be passed as an object, and when the server is launched through `bin/shim.js` we populate config from env vars

Reply:
We could split `config.js` into `env.js` and `config.js` where `env.js` only has extraction from `process.env` and it's config that has an exported class that allows constructor overrides, wdyt?

Reply:
I gave this a shot, but quickly ran into limitations of the current setup. I'm going to revisit this after https://github.com/filecoin-saturn/L1-node/pull/52, when L1 was turned into a class.